client/conn_test.go

```diff
@@ -0,0 +1,126 @@
+package client
+
+import (
+	"fmt"
+
+	. "github.com/pingcap/check"
+
+	"github.com/go-mysql-org/go-mysql/mysql"
+)
+
+type connTestSuite struct {
+	c    *Conn
+	port string
+}
+
+func (s *connTestSuite) SetUpSuite(c *C) {
+	var err error
+	addr := fmt.Sprintf("%s:%s", *testHost, s.port)
+	s.c, err = Connect(addr, *testUser, *testPassword, "")
+	if err != nil {
+		c.Fatal(err)
+	}
+
+	_, err = s.c.Execute("CREATE DATABASE IF NOT EXISTS " + *testDB)
+	c.Assert(err, IsNil)
+
+	_, err = s.c.Execute("USE " + *testDB)
+	c.Assert(err, IsNil)
+
+	s.testExecute_CreateTable(c)
+}
+
+func (s *connTestSuite) TearDownSuite(c *C) {
+	if s.c == nil {
+		return
+	}
+
+	s.testExecute_DropTable(c)
+
+	if s.c != nil {
+		s.c.Close()
+	}
+}
+
+var (
+	testExecuteSelectStreamingRows      = [...]string{"foo", "helloworld", "bar", "", "spam"}
+	testExecuteSelectStreamingTablename = "execute_plain_table"
+)
+
+func (s *connTestSuite) testExecute_CreateTable(c *C) {
```

Comment:
not sure if we should follow go's naming style like `testExecuteCreateTable`. This repo has some variables using underscore 🤔

Reply:
I copypasted this style from neighbour clientTestSuite :)